packages/amplify_authenticator/lib/src/models/authenticator_builder.dart

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The diff here is misleading. This is a new file. The file on the left was renamed `authenticator_step.dart`.